packages/wdio-junit-reporter/src/index.ts

```diff
@@ -25,6 +25,11 @@ class JunitReporter extends WDIOReporter {
             : /[^a-zA-Z0-9@]+/ // Reason for ignoring @ is; reporters like wdio-report-portal will fetch the tags from testcase name given as @foo @bar
     }
 
+    private _ansiRegex = new RegExp([
```

Comment:
No need to have it as class property either, I suggest to put it at the top

```js
const ansiRegex = new RegExp([...
```

Reply:
ok

Reply:
It fails node 12 on the top 😩